### ISSUE TITLE
Rebalance model to better distribute scores?

### ISSUE THREAD

**Issue by mattfogel** [fuzzy-io/personalhunt#25]
Could it make sense to rebalance the base models for better score distribution? Everything on my top list seems to be falling between 3.5 and 5.

**Comment by evanp** [comment]
Yeah, it'd make sense. I'll see what I can do, but I don't think we'll get anything done before Tuesday.
